packages/flutter/lib/src/material/button.dart

```diff
@@ -195,14 +195,27 @@ class MaterialButton extends StatefulWidget {
   /// Defaults to the highlight color from the [Theme].
   final Color highlightColor;
 
-  /// The z-coordinate at which to place this button.
+  /// The z-coordinate at which to place this button. This controls the size of
+  /// the shadow below the button.
   ///
   /// Defaults to 0.
+  ///
+  /// See also:
+  ///
+  ///  * [FlatButton], a material button specialised for the case where the
```

Comment:
I think specialised is the UK spelling. I think we're going to just live with UK spellings :-)